blocks/author-profile/author-profile.js

```diff
@@ -0,0 +1,59 @@
+import {
+  createOptimizedPicture, decorateIcons,
+  getMetadata,
+} from '../../scripts/aem.js';
+import {
+  div, h2, p, a, span,
+} from '../../scripts/dom-builder.js';
+import ffetch from '../../scripts/ffetch.js';
+
+const breakpoints = [
+  { width: '120' },
+];
+
+function extractAuthorDescription(details) {
+  let { description } = details;
+  if (!description) {
+    if (details.title) {
+      description = details.title;
+      if (details.author && details.title.startsWith(`${details.author}, `)) {
+        description = details.title.substring(details.author.length + 2);
+      }
+    } else {
+      description = 'Author';
+    }
+  }
+  return description;
+}
+
+async function getAuthorDetails() {
+  // #todo: handle multiple authors
+  const name = getMetadata('author');
+  const result = name
+    ? await ffetch('/authors-index.json').filter((entry) => entry.author === name).limit(1).all() : null;
```

Comment:
in other projects I have seen that indexes that are expected to be fetched multiple times are added to document upon first fetch and then reused. is that worth a consideration here @mhaack ?

Reply:
You mean sort of pre-fetch? I would only do this if we see perf issues.

Reply:
I mean cache on first request, but I agree to your point 👍

Reply:
It is already cached on first request by the browser.